communicationmanager.go

```diff
@@ -232,7 +239,7 @@ func (cm *communicationManager) close() {
 		cm.alerts.Close()
 	}
 
-	// Close crypto context
+	// Close crypto handler
```

Comment:
please move closing of crypto handler and  cryptoContext after closing qmqp

Reply:
Done